dev/release/post-09-docs.sh

```diff
@@ -43,22 +53,39 @@ git checkout -b asf-site origin/asf-site
 git rebase apache/asf-site
 git branch -D ${branch_name} || :
 git checkout -b ${branch_name}
+# list and remove previous versioned docs
 versioned_paths=()
 for versioned_path in docs/*.0/; do
   versioned_paths+=(${versioned_path})
+  rm -rf ${versioned_path}
 done
+# add to list and remove dev docs
+versioned_paths+=("docs/dev/")
+rm -rf docs/dev/
+if [ "$is_major_release" = true ] ; then
+  # copy the current stable docs to temporary directory
+  # (remove java reference to reduce size)
+  rm -rf docs/java/reference/
+  cp -r docs/ docs_temp/
+fi
+# delete current stable docs and restore all previous versioned docs
 rm -rf docs/*
 git checkout "${versioned_paths[@]}"
 curl \
   --fail \
   --location \
   --remote-name \
   https://apache.jfrog.io/artifactory/arrow/docs/${version}/docs.tar.gz
+cp ../docs.tar.gz .
```

Comment:
Is this needed?
It seems that `../docs.tar.gz` doesn't exist.

Reply:
Whoops, that was from local testing (to not download the archive on every test run), will remove.